tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -205,6 +205,13 @@ tests:
             CONFIG_DT_HAS_NXP_FLEXIO_SPI_ENABLED
     platform_allow:
       - mimxrt1170_evk/mimxrt1176/cm7
+  drivers.spi.mimxrt1180_evk_flexio_spi.loopback:
+    extra_args: DTC_OVERLAY_FILE="boards/mimxrt1180_evk_flexio_spi.overlay"
```

Comment:
you can update the drivers.spi.mimxrt1170_evk_mimxrt1176_cm7_flexio_spi.loopback and merge them with rt1180 with below
```
  drivers.spi.flexio_spi.loopback:
    extra_args:
     - platform:imxrt1170_evk/mimxrt1176/cm7:DTC_OVERLAY_FILE="boards/mimxrt1170_evk_mimxrt1176_cm7_flexio_spi.overlay"
       - platform:mimxrt1180_evk/mimxrt1189/cm33:DTC_OVERLAY_FILE="boards/mimxrt1180_evk_flexio_spi.overlay"
       - platform:mimxrt1180_evk/mimxrt1189/cm7:DTC_OVERLAY_FILE="boards/mimxrt1180_evk_flexio_spi.overlay"

    filter: CONFIG_DT_HAS_NXP_FLEXIO_ENABLED and
            CONFIG_DT_HAS_NXP_FLEXIO_SPI_ENABLED
    platform_allow:
      - mimxrt1170_evk/mimxrt1176/cm7
      - mimxrt1180_evk/mimxrt1189/cm33
      - mimxrt1180_evk/mimxrt1189/cm7
```